proto/eventcounter/timeseries.proto

```diff
@@ -23,6 +23,11 @@ message VariationTimeseries {
 }
 
 message Timeseries {
+  enum Unit {
```

Comment:
I think that it'll be better to define this filed in GetEvaluationTimeseriesCountResponse. What do you think?